event.go

```diff
@@ -46,6 +49,32 @@ func NewEvent(t EventType, o Identifiable) *Event {
 	return NewEventWithEncoding(t, o, EncodingTypeMSGPACK)
 }
 
+// NewErrorEvent returns a new (error) Event embedded with the provided elemental.Error
+func NewErrorEvent(ee Error, encoding EncodingType) *Event {
+
+	data, err := Encode(encoding, ee)
+	if err != nil {
+		panic(fmt.Sprintf("unable to create new error event: %s", err))
+	}
+
+	event := &Event{
+		Type: EventError,
+		// TODO: should we even populate this field at all? is it sufficient that the client can assume they should just
+		// deserialize this into an elemental error since the event type is error?
+		Identity:  string(EventError),
```

Comment:
@primalmotion

Reply:
this field should not be populated. EventError is not an identity per se anyway

Reply:
🔥 

![image](https://user-images.githubusercontent.com/1800781/74785925-5d11b100-5260-11ea-9925-73bfc4fd410a.png)